Add instructions on how to verify sources for PMC members (#101)
apache/airflow-client-python: dev/README_RELEASE_CLIENT.md

## Signature check
See Airflow process documented [here](https://github.com/apache/airflow/blob/master/dev/README_RELEASE_AIRFLOW.md#signature-check).

## Sources check

> The code of the Python Client is generated using Open API generator and the generated code is committed to

Reviewer: ```suggestion
The code of the Python Client is generated using OpenAPII generator and the generated code is committed to
```

the repository, therefore the Source code check should consist of two steps:

1. Checkout the release tag of the "airflow-client-python"
2. Follow the [Release process](#release-process) above and generate client's code above

> 3. Run ``git diff`` to see the differences - review them. Generally the code generated should only contain

Reviewer: ```suggestion
3. Run ``git diff`` to see the differences - review them. Generally, the code generated should only contain
```

   small differences regarding authentication.
4. git reset --hard <RELEASE_TAG>

> Compare the sources with the packaged sources following the process describd in Airflow:

Reviewer: ```suggestion
Compare the sources with the packaged sources following the process described in Airflow:
```
